src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>ActivityFeed</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+	<nav>
```

Comment:
There's a tab here, and on some lines below, so you should do a find/replace to turn tabs into two spaces.